src/main/java/attendance/util/DateFormatUtil.java

```diff
@@ -1,31 +1,22 @@
 package attendance.util;
 
-import java.time.LocalDateTime;
+import static attendance.error.ErrorMessage.ERROR_INVALID_TIME;
+
 import java.time.LocalTime;
 import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeParseException;
 
 public class DateFormatUtil {
+    public final static DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("MM월 dd일 EEEE");
```

Comment:
modifier 순서를 지켜주세요~

```suggestion
    public static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("MM월 dd일 EEEE");
```

https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers

Reply:
해당 내용 전반적으로 체크해주세요!

Reply:
수정했습니다..! 한 번 잘못사용하니 자각 없이 계속 잘못되게 선언했었군요.. 그 밖의 선언자 순서에 대해서도 다시 한 번 공부할 수 있었습니다. 감사합니다!